config/rinkeby.exs

```diff
@@ -0,0 +1,82 @@
+use Mix.Config
```

Comment:
How do you feel about renaming this file to `config/testnet.exs`? I could see this being useful for testing other chains.

Reply:
no problem